### PR TITLE
Nc news 3

### PR DESCRIPTION
**Pull request by amy-baker**

### CODE REVIEW

File: __tests__/app.test.js
Comment (nc-poppy-m) [inline __tests__/app.test.js]: Remember to remove console.logs and commented out code before pushing your code

File: db/model.js
Comment (nc-poppy-m) [inline db/model.js]: Try and be consistent - in your previous model you use a .then block so it's good practice to use the same syntax :)

File: db/model.js
Comment (nc-poppy-m) [inline db/model.js]: As we are not interacting with the database, this should be handled in the controller

File: __tests__/app.test.js
Comment (nc-poppy-m) [inline __tests__/app.test.js:43]: This test will never fail, so when you write a new endpoint, how could you check this has been updated?

**Review by nc-poppy-m — CHANGES_REQUESTED** [review]
Just a few changes to make before merging - once you've made your changes, put in another pr tomorrow and we'll have another look 😄

**Review by mezzDavies — APPROVED** [review]
Approved 😄 IF you wanted to refactor you could simply require the endpoint.json file into the controller file as well as the test file (without fsread) and assert that what is sent back from the endpoint.controller is equal to the file in the tests.

**Pull request merged**
